frontend/src/components/podcast/helpers/get-podcast-periodicity/get-podcast-periodicity.helper.ts

```diff
@@ -0,0 +1,70 @@
+import { DateFormatType } from 'common/enums/enums';
+import { Episode } from 'common/types/types';
+import {
+  getDifferenceInDays,
+  getDifferenceInWeeks,
+  getDifferenceInMonths,
+  getFormattedDate,
+} from 'helpers/helpers';
+
+const getPodcastPeriodicity = (episodes: Episode[]): string => {
+  if (!episodes.length) {
+    return 'No data yet';
+  }
+
+  const episodesDatesWithoutTime = episodes.map((episode) =>
+    getFormattedDate(new Date(episode.createdAt), DateFormatType.ISO_DATE_000Z),
+  );
+  const uniqueSortedEpisodesDates = [...new Set(episodesDatesWithoutTime)].sort(
+    (prev, next) => new Date(prev).getTime() - new Date(next).getTime(),
+  );
+
+  const firstDate = uniqueSortedEpisodesDates[0];
```

Comment:
What does it mean `0`? Let's use [destructuring assignment](https://javascript.info/destructuring-assignment)
```suggestion
  const [firstDate] = uniqueSortedEpisodesDates;
```